src/main/resources/postman-v2/item.mustache

```diff
@@ -5,6 +5,13 @@
                         {{#vendorExtensions.postmanRequests}}
                             {
                                 "name": "{{{name}}}",
+                                "response": [{
+                                    "name": "{{response.name}}",
+                                    "code": "{{response.code}}",
```

Comment:
@jlengrand code should be `integer` (Postman JSON schema v2.1.0)

Reply:
Good point!